<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenCVCamera</class>
 <widget class="QDialog" name="OpenCVCamera">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>939</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_17">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <widget class="QWidget" name="viewfinderPage">
        <widget class="QLabel" name="viewfinder">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>641</width>
           <height>521</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="displayImage">
         <property name="geometry">
          <rect>
           <x>660</x>
           <y>60</y>
           <width>251</width>
           <height>201</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget_4">
         <property name="geometry">
          <rect>
           <x>660</x>
           <y>10</y>
           <width>251</width>
           <height>41</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Select Camera:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="selectDeviceCB">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="">
         <property name="geometry">
          <rect>
           <x>661</x>
           <y>267</y>
           <width>251</width>
           <height>259</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Properties</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QCheckBox" name="checkBox_enableSaveImg">
               <property name="text">
                <string>Enable Save Image</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_14">
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Resolution </string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="cb_resolution"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_15">
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>Image Format</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="cb_formats">
                 <property name="cursor">
                  <cursorShape>UpArrowCursor</cursorShape>
                 </property>
                 <property name="editable">
                  <bool>false</bool>
                 </property>
                 <property name="duplicatesEnabled">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLabel" name="lbl_save">
               <property name="font">
                <font>
                 <italic>true</italic>
                </font>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QPushButton" name="btnCapturePhoto">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>40</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="text">
               <string> Capture Photo</string>
              </property>
              <property name="icon">
               <iconset>
                <normalon>D:/Nestor/QT/QTAppsOpenCV-V2/images/cam_icon.png</normalon>
                <activeon>D:/Nestor/QT/QTAppsOpenCV-V2/images/cam_icon.png</activeon>
                <selectedon>D:/Nestor/QT/QTAppsOpenCV-V2/images/cam_icon.png</selectedon>
               </iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnProcess">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>40</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Process</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>D:/Nestor/QT/QTAppsOpenCV-V2/images/905px-Noun_Project_process_icon_1261773_cc.svg.png</normaloff>D:/Nestor/QT/QTAppsOpenCV-V2/images/905px-Noun_Project_process_icon_1261773_cc.svg.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>100</width>
                <height>30</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="previewPage"/>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
